.travis.yml

```diff
@@ -552,6 +552,22 @@ osx_rust_tests: &osx_rust_tests
       PATH="${PYENV_ROOT}/versions/${PYENV_PY36_VERSION}/bin:${PATH}"
     - CACHE_NAME=macosrusttests
 
+# -------------------------------------------------------------------------
+# Remote integration tests
+# -------------------------------------------------------------------------
+
+remote_integration_tests: &remote_integration_tests
+  <<: *py36_linux_test_config
+  name: "Integration tests with mock remote stack"
+  env:
+    - *py36_linux_test_config_env
+    - CACHE_NAME=remote_integration_tests
+  before_script:
+    - ./build-support/bin/native/cargo.sh build --manifest-path src/rust/engine/Cargo.toml --release --bin local_cas --bin local_execution_server
```

Comment:
I recommend a dedicated script to run this command, like `bootstrap_mock_remote_server.sh` (or `.py`).